.github/workflows/busola-backend-build.yml

```diff
@@ -0,0 +1,40 @@
+name: Busola Backend Build
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - ".github/workflows/busola-backend-build.yml"
+      - "backend/**"
+      - "public/**"
+      - "src/**"
+      - "tests/**"
+      - "package.json"
+      - "Makefile"
+      - "Dockerfile"
+  pull_request_target:
+    types: [opened, edited, synchronize, reopened, ready_for_review]
+    paths:
+      - ".github/workflows/busola-backend-build.yml"
+      - "backend/**"
+      - "public/**"
+      - "src/**"
+      - "tests/**"
+      - "package.json"
+      - "Makefile"
+      - "Dockerfile"
```

Comment:
paths should be the same as `run_if_changed` in the previous prow flow
https://github.com/kyma-project/test-infra/blob/main/prow/jobs/kyma-project/busola/busola-backend/busola-backend.yaml#L12